templates/patterns/modal/modal-language.html.twig

```diff
@@ -1,89 +1 @@
-{% spaceless %}
-{# Parameters:
-  - modal
-    - id
-    - size
-    - header
-      - title
-      - label
-    - body
-      - links
-    - footer
-      - label
-#}
-{% if header %}
-  {% set _header %}
-    <h5 class="modal-title" id="languageeModalLabel">{{ header.title|default('Select your language'|t) }}</h5>
-    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="{{ header.label|default('Close'|t) }}"></button>
-  {% endset %}
-{% endif %}
-
-{% if eu_links or non_eu_links %}
-  {% set _body %}
-    <div class="container">
-      <div class="row">
-        <div class="col col-lg-8 offset-lg-2">
-          <h5 class="bcl-language__title">{{ 'EU official languages'|t }}</h5>
-        </div>
-      </div>
-      <div class="row">
-      {% set _number_items = (eu_links|length / 2) %}
-      {% for link in eu_links|batch(_number_items) %}
-        {% if loop.index % 2 == 0 %}
-          <div class="col col-lg-4 offset-lg-2">
-        {% else %}
-          <div class="col col-lg-4">
-        {% endif %}
-            <div class="bcl-language__list">
-              {% for id, data in link %}
-                <a id="{{ data.id }}" href="{{ data.href|default('#') }}" class="bcl-language__item">{{ data.name }}</a>
-              {% endfor %}
-            </div>
-          </div>
-      {% endfor %}
-      </div>
-      {% if non_eu_links %}
-          <div class="row">
-            <div class="col col-lg-8 offset-lg-2">
-              <h5 class="bcl-language__title">{{ 'Non EU languages'|t }}</h5>
-            </div>
-          </div>
-        <div class="row">
-        {% set _number_items = (non_eu_links|length / 2) %}
-        {% for link in non_eu_links|batch(_number_items) %}
-          {% if loop.index % 2 == 0 %}
-            <div class="col col-lg-4 offset-lg-2">
-          {% else %}
-            <div class="col col-lg-4">
-          {% endif %}
-              <div class="bcl-language__list">
-                {% for id, data in link %}
-                  <a  id="{{ data.id }}" href="{{ data.href|default('#') }}" class="bcl-language__item">{{ data.name }}</a>
-                {% endfor %}
-              </div>
-            </div>
-        {% endfor %}
-        </div>
-      {% endif %}
-    </div>
-  {% endset %}
-{% endif %}
-
-{% if footer %}
-  {% set _footer %}
-    <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">{{ header.label|default('Close'|t) }}</button>
-  {% endset %}
-{% endif %}
-
-{% set _data = {
-  id: id,
-  size: size,
-  header: _header,
-  body: _body,
-  footer: _footer,
-} %}
-
-<div class="bcl-language">
-  {% include '@oe-bcl/bcl-modal/modal.html.twig' with _data only %}
-</div>
-{% endspaceless %}
+{% include '@oe-bcl/bcl-language-list/language-list.html.twig' with language_list only %}
```

Comment:
this is very nice, but the close buttons are gone, and there is no modal title

Reply:
![image](https://user-images.githubusercontent.com/6751237/158992667-ecfa72ee-bb2f-4e8c-86d2-8c5966d851fd.png)

@drishu  Title and close button are there. It's true than in the netlify there is a margin top and the title is not so close to the top. I'm going to take a look. Thanks.